behave/docker-compose.yml

```diff
@@ -148,6 +162,29 @@ services:
       STATIC_FILES_BACKEND: local
       ADMIN_PASSWORD: cla_admin
       ALLOWED_HOSTS: "*"
+      NON_ROTA_START_TIME_HR: 1
+      NON_ROTA_END_TIME_HR: 22
+      EDUCATION_START_TIME_HR: 1
+      EDUCATION_END_TIME_HR: 22
+      DISCRIMINATION_START_TIME_HR: 1
+      DISCRIMINATION_END_TIME_HR: 22
+
+  clabackendooh:
+    <<: *clabackend
+    ports:
+      - "8014:8000"
+    depends_on:
+      - clabackend
+    container_name:
+      clabackendooh
+    environment:
+      <<: *clabackendenvironment
+      NON_ROTA_START_TIME_HR: 22
```

Comment:
Is there an option for this to be in hours when there is an hour between start and end? Can we make it so they are the same?

Reply:
There is no way around this without re-writing CLA_Backend as the datetime is preset to accept 0 minutes so we cannot make the end time 23:58 to 23:59, 23 is the latest as datetime won't accept 24

Reply:
Make this set be an hour in the middle of the night (0 to 1 or perhaps 1 till 2) - the least likely time for the test to be run.